src/main/java/mate/academy/hibernate/relations/service/impl/ActorServiceImpl.java

```diff
@@ -1,16 +1,25 @@
 package mate.academy.hibernate.relations.service.impl;
 
+import mate.academy.hibernate.relations.dao.ActorDao;
+import mate.academy.hibernate.relations.dao.impl.ActorDaoImpl;
 import mate.academy.hibernate.relations.model.Actor;
 import mate.academy.hibernate.relations.service.ActorService;
+import org.hibernate.SessionFactory;
 
 public class ActorServiceImpl implements ActorService {
+    private ActorDao actorDao;
+
+    public ActorServiceImpl(SessionFactory sessionFactory) {
+        this.actorDao = new ActorDaoImpl(sessionFactory);
+    }
+
     @Override
     public Actor add(Actor actor) {
-        return null;
+        return actorDao.add(actor);
     }
 
     @Override
     public Actor get(Long id) {
-        return null;
+        return actorDao.get(id).orElse(new Actor());
```

Comment:
```suggestion
        return actorDao.get(id).orElseThrow(() -> RuntimeException("Couldn't find actor by id : " + id));
```
Should return error, if return default object, there would be no information about error
Especially when getting entity by id.
ps The same with other get(id) methods

Reply:
ok

Reply:
Maybe in future someone should cover this issue with tests.

Reply:
Wait. We already throw an exception on dao layer in case of unsuccessful get operation, why repeat it here? We can have an issue here only when we have trouble getting object from DB in dao, so we simply will not get to this method because get() in dao will throw our custom made DataProcessiingException, which even have additional info about object's id!
Anyway, i think throwing an exception here in proposed style is redundant.

Reply:
You wrote: "Should return error, if return default object, there would be no information about error"
Yes we will have information, because, as i said, get() in dao layer throws custom error with additional information about object's id!

Reply:
Good point))
return actorDao.get(id).get()
So here use method get()

Reply:
"So here use method get()" - exactly!
Thanks!